src/plugins/intel_gna/src/frontend/quantization.hpp

```diff
@@ -81,7 +81,7 @@ inline T SaturationCast(float value, uint32_t* saturation_counter = nullptr) {
 /**
  * @brief Apply FQ levels onto a value
  */
-float ApplyFQ(float value, float input_low, float input_high, float output_low, float output_high, uint32_t levels);
+float ApplyFQ(float value, float input_low, float input_high, float output_low, float output_high, size_t levels);
```

Comment:
do you need size_t to uint32_t because of some specific reason or because you changed argument type used for invocation of the method from auto to size_t?